emission/analysis/config.py

```diff
@@ -1,11 +1,17 @@
 import json
+import os
 
 def get_config_data():
     try:
+        print("Trying to open debug.conf.json")
         config_file = open('conf/analysis/debug.conf.json')
```

Comment:
I am not fully convinced that this is the right approach. If we keep this backwards compat code around forever, we don't have any motivation for people to change to the new structure in the future. Having said that, I will not hold up the merge for this, but I do want to make sure that we have a plan to remove this in a year or so, and to notify users to change their config files, so that we don't end up with a bunch of hacky backwards compat code strewn all around the codebase.